ios/Runner/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+<key>NSLocationWhenInUseUsageDescription</key>
+<string>This app needs location access to determine your Wi-Fi network</string>
+<key>NSLocationAlwaysUsageDescription</key>
+<string>This app needs location access to determine your Wi-Fi network</string>
```

Comment:
What has this to do with Wifi ?

Reply:
I have to see more thoroughly. What I found online were conflicting views. Apple wont allow you to see wifi because of some notorious privacy policies. Other one said that if you wanna see wifi it(connectivity package)should be fine. I can check and get something more promising